playground/react/App.tsx

```diff
@@ -1,14 +1,22 @@
-import { StoryblokComponent, useStoryblok } from '@storyblok/react';
+import { StoryblokRichText, useStoryblok } from '@storyblok/react';
 import React from 'react';
 
 function App() {
-  const story = useStoryblok('home', { version: 'draft' });
+  // const story = useStoryblok('home', { version: 'draft' });
+  const story = useStoryblok('react/test-richtext', { version: 'draft' });
```

Comment:
is this a leftover? or should we just leave this and remove the comment above?

Reply:
@edodusi I left them there so you could test them, I just committed back the original playground without the richtext